src/binary-reader-objdump.cc

```diff
@@ -157,19 +167,47 @@ class BinaryReaderObjdumpPrepass : public BinaryReaderObjdumpBase {
  public:
   using BinaryReaderObjdumpBase::BinaryReaderObjdumpBase;
 
-  Result OnFunctionName(Index function_index,
-                        string_view function_name) override;
+  Result OnFunctionName(Index index,
```

Comment:
nit: I generally haven't been defining functions like this inline in the class body. Any reason to prefer it this way?

Reply:
My thinking was because (a) these are all internal to the cpp file (b) they are both small forwarding functions (c) it makes the purpose of this class and these methods clear simply by ready the class def.

Whats wrong with inlining short methods like these?